adafruit_ws2801.py

```diff
@@ -49,7 +54,9 @@ class WS2801:
             time.sleep(2)
     """
 
-    def __init__(self, clock, data, n, *, brightness=1.0, auto_write=True):
+    def __init__(
+        self, clock, data, n: int, *, brightness: float = 1.0, auto_write: bool = True
```

Comment:
I think `clock` and `data` can be typed with `microcontroller.Pin`. It will take one extra import that can go inside of the try block underneath the existing typing import.

The rest of these changes look good to me.

Reply:
Done!